modules/virt-measuring-latency-vm-secondary-network.adoc

```diff
@@ -0,0 +1,229 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc
+
+:_content-type: PROCEDURE
+[id="virt-measuring-latency-vm-secondary-network_{context}"]
+= Verifying connectivity and measuring latency between virtual machines on a secondary network
+
+As a cluster administrator, you use a predefined checkup to verify network connectivity and measure latency between virtual machines (VMs) that are attached to a secondary network interface. Set up the namespace and service account for the framework; create the `ClusterRole` and `ClusterRoleBinding` objects for the service account; enable permissions for the checkup; create the input config map; and run the checkup. After you record the results of the checkup, delete all the framework resources that you previously created.
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`).
+* You logged in to the cluster as a user with the `cluster-admin` role.
+* The cluster has at least two worker nodes.
+* The Multus Container Network Interface (CNI) plug-in is installed on the cluster.
+* You configured a Linux bridge network attachment definition for a namespace.
```

Comment:
> Linux bridge

We are suppose to support any network attachment definition (that CNV supports), so I think we should not mention a specific one here. The NAD is expected to be already defined in the cluster.

Reply:
I agree. Since the user has to create a NetworkAttachmentDefinition (as well as a NodeNetworkConfigurationPolicy) when setting up a secondary network, you do not have to mention it here as a prerequisite. If they haven't done so, their network won't work. :-)

I think you could leave out the first two prerequisites as well. I noticed that in our docs, we assume that the user has 'oc' installed if they are using a CLI procedure. Also, pretty much every task involving networks requires a cluster admin.

Reply:
I don't think you should mention that the user needs a Linux Bridge NetworkAttachmentDefinition. But I do think it's worth mentioning that a NetworkAttachmentDefinition is expected (whichever type the user deploys). 
From a user's perspective, it seems to that without these lines (current commit), I can count on the checkup-framework to create a dedicated NetworkAttachmentDefinition